### ISSUE TITLE
Changing line width at different points of the connector (or at least at line start and line end)

### ISSUE THREAD

**Issue by AlexDaniel** [jgraph/drawio#2811]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

See this the modern edition of the well-known [Minard's map](https://en.wikipedia.org/wiki/Charles_Joseph_Minard#The_map_of_Napoleon's_Russian_campaign) ([original](https://user-images.githubusercontent.com/5507503/167685373-3cbfe8a6-4d94-4142-8004-d04cb3dfca89.jpg)):

![image](https://user-images.githubusercontent.com/5507503/167684729-8f6201f0-5946-4130-8439-6473526bffc7.png)

The graph relies heavily on the ability to change arrow/line width to indicate the size of the remaining army. I find myself often in the same situation – I want to visually indicate on the graph not just how something is connected, but also the amount that is flowing through and how it changes.

The only alternative I know is using this shape:
![image](https://user-images.githubusercontent.com/5507503/167686315-44ffc6a4-2e86-44d0-ba7a-d8847c0b52de.png)
But it is not convenient because of its 90° nature, and the start of the arrow is always from 0 width (which is not always the case). Another option is to use a custom polygon, but that is very difficult.

Basically, all I need in most cases is simply a connector where both line start width and line end width can be set to different values.
